src/main/webapp/app/overview/exercise-details/request-feedback-button/request-feedback-button.component.ts

```
@@ -0,0 +1,124 @@
+            this.athenaEnabled = profileInfo.activeProfiles?.includes(PROFILE_ATHENA);
+        });
+        this.isExamExercise = isExamExercise(this.exercise());
+        if (this.isExamExercise || !this.exercise().id) {
+            return;
+        }
+        this.updateParticipation();
+    }
+
+    private updateParticipation() {
+        if (this.exercise().id) {
+            this.exerciseService.getExerciseDetails(this.exercise().id!).subscribe({
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)